native/cocos/audio/android/AudioPlayerProvider.h

```diff
@@ -92,7 +92,6 @@ class AudioPlayerProvider {
 
     bool isSmallFile(const AudioFileInfo &info);
 
-private:
```

Comment:
Why remove the access modifiers ?

Reply:
It's already privates, clang-tidy warns it